script.js

```diff
@@ -1,4 +1,79 @@
+//include current gamemode
+var currentGameMode = 'insert username to begin';
```

Comment:
Dun actually need this variable. Can test if userName=="" instead to decide if should assign input as userName instead of using it as input to play scissors paper stone.